server/apiserver/apiserver.go

```diff
@@ -66,6 +66,24 @@ func (api *ApiServer) UpdatePixel(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (api *ApiServer) GetLastUserModification(user_id string) string{
+
+	// Testing with some dummy data
+	m := consensus.BackendMessage{ Type: consensus.GET_LAST_USER_UPDATE, Data: fmt.Sprintf("get %s", user_id)}
```

Comment:
nit: rather than storing the data in a string can we refactor our existing code to use something like this https://blog.golang.org/gobs-of-data